pkg/mkimage-raw-efi/install

```diff
@@ -245,8 +252,21 @@ if [ "$INSTALL_DEV" != "$INSTALL_PERSIST" ]; then
            fi
         fi
      done
-     if [ $DISKS_TO_MERGE_COUNT -eq 0 ]; then
-          # in case of only comma provided
+
+     if [ "$SKIP_ZFS_CHECKS" = false ]; then
+        # Do ZFS minimum requirements checks (64GB memory and 3 physical drives in INSTALL_PERSIST)
+        # If requirements are not met, fallback to ext4 filesystem.
+        system_memory_GB="$(/usr/bin/free -g | grep Mem | awk '{print $2}')"
+
+        if [ "$system_memory_GB" -lt 64 ] || [ $DISKS_TO_MERGE_COUNT -lt 3 ]; then
+           echo "WARNING: ZFS installation minimum requirements are not met, installing ext4 filesystem instead."
+           # The following setting will install ext4 persist
+           INSTALL_ZFS=false
+        fi
+     fi
+
+     if [ $DISKS_TO_MERGE_COUNT -eq 0 ] || [ "$INSTALL_ZFS" = false ]; then
```

Comment:
Seems we need more complex logic here. If we define a comma but don't meet the requirements (and they're not explicitly omitted), we should adjust logic for ext4 to use P3 on the same disk, something like that:
```
     if [ $DISKS_TO_MERGE_COUNT -eq 0 ]; then
        MAKE_RAW_PARTS="efi imga imgb conf persist"
        if [ "$INSTALL_ZFS" = true ]; then
           DISK_WITH_P3="$INSTALL_DEV"
           POOL_CREATION_COMMAND_SUFFIX=$P3_ON_BOOT_PLACEHOLDER
           DISKS_TO_MERGE_COUNT=1     
        fi
     fi
```

Reply:
I did not understand your comment. Whenever DISKS_TO_MERGE_COUNT=0 we pick ext4 by default irrespective
of  INSTALL_ZFS is true or false.

My code at line 268 is for scenario where customer provided multiple disks and system has only 16GB memory, then we install ext4. because memory requirement is not met (INSTALL_ZFS will be false)

Reply:
We install zfs onto partition of the disk (i.e. sda9) here in case of DISKS_TO_MERGE_COUNT=0.

Reply:
Sorry, it is my mistake, your current approach cover option when we do not install zfs and will go into the check over OR branch and correctly set MAKE_RAW_PARTS.

Reply:
No worries 👍